src/app/globals.css

```diff
@@ -1,13 +1,41 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@font-face {
+  font-family: 'RobotoCondensed';
+  src: url('assets/fonts/RobotoCondensed-Regular.ttf');
+  font-weight: 400;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'DancingScript';
+  src: url('assets/fonts/DancingScript-Regular.ttf');
+  font-weight: 400;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-accent: #f25f5c;
+  --secondary-accent: #3c6e71;
+  --tertiary-accent: #bfdbf7;
+  --background-gradient: linear-gradient(
+    170deg,
+    rgba(40, 75, 99, 1) 0%,
+    rgba(60, 110, 113, 1) 100%
+  );
+  --primary-background--tertiary-font: #fff;
+  --secondary--background--primary-font: #737373;
```

Comment:
I get the reasoning behind the double name, but I also long for a different solution :D

Reply:
resolved in [f3054ea](https://github.com/Seborider/plugs./pull/18/commits/f3054ead66ae8209d931bec1ca3896069475ef8b)